### PR TITLE
Update nudge.md

### PR DESCRIPTION
**Pull request by StephanieTW**
## 🖼 Context

Updated the interaction section so it matches what we agreed for mobile, web, and mobile web.

## 🚀 Changes

`The whole component is tappable — it triggers the link. But tapping the area around the close icon dismisses the nudge.`

### CODE REVIEW

**Comment by gusso** [comment]
I think this implies (not too strongly, of course) that tapping the area around the close icon will trigger both the link and the closing. It might be clearer if there was a word like "except" somewhere to clarify it's one or the other, what do you think?

**Comment by StephanieTW** [comment]
Yes, good point. Just heading in to a meeting but I'll rework this after to include 'except'

File: src/components/nudge.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ Place a nudge at the top of the page or section it applies to, or below a sectio
 
 Nudges direct users to another view to complete an action. Because the content they contain is optional, nudges are always dismissible.
 
-On mobile, the whole component should be tappable. 
+The whole component is tappable — it triggers the link. But tapping the area around the close icon dismisses the nudge.
```
Comment (StephanieTW) [inline src/components/nudge.md:55]:
```suggestion
Tapping anywhere on the component triggers the link — except for the area around the close icon, which dismisses the nudge.
```

**Comment by StephanieTW** [comment]
How about we update it to this:

Tapping anywhere on the component triggers the link — except for the area around the close icon, which dismisses the nudge.

**Review by gusso — APPROVED** [review]

**Pull request merged**